src/VirtualClient/VirtualClient.Actions/SPECjvm/SpecJvmExecutor.cs

```diff
@@ -115,6 +116,24 @@ protected override async Task InitializeAsync(EventContext telemetryContext, Can
             this.javaExecutableDirectory = javaExecutable.Metadata[PackageMetadata.ExecutablePath].ToString();
         }
 
+        /// <summary>
+        /// Returns true/false whether the component is supported on the current
+        /// OS platform and CPU architecture.
+        /// </summary>
+        protected override bool IsSupported()
+        {
+            bool isSupported = base.IsSupported()
+                && (this.Platform == PlatformID.Win32NT || this.Platform == PlatformID.Unix)
+                && (this.CpuArchitecture == Architecture.X64 || this.CpuArchitecture == Architecture.Arm64);
+
+            if (!isSupported)
+            {
+                this.Logger.LogNotSupported("SpecJvm", this.Platform, this.CpuArchitecture, EventContext.Persisted());
```

Comment:
maybe SPECjvm

Reply:
somehow i am not able to push in this branch. I will make this change along with my next PR